Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN yum repolist && \
         python-dateutil \
         subprocess32 \
         psutil \
+        kubernetes \
```

Comment:
Unrelated to this PR: Personally I would prefer to pin those to a specific version to reduce the risk of surprise.